pkg/logql/log/filter.go

```diff
@@ -421,6 +421,9 @@ func (l equalFilter) String() string {
 }
 
 func newEqualFilter(match []byte, caseInsensitive bool) MatcherFilterer {
+	if caseInsensitive {
```

Comment:
cc @MasslessParticle contains expects lowered values.

Not sure why we use contains for equality match btw.